jest.config.js

```diff
@@ -0,0 +1,23 @@
+const nextJest = require('next/jest')
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
```

Comment:
I think this should be good to remove, yeah?

Reply:
The comment "provide the path..."?

Reply:
Yeah...I'm wondering what the point of this is?